### PR TITLE
Extend user.name mapping to Windows Integration Package

### PR DESCRIPTION
**Pull request by ChriZzn**
## Proposed commit message
Adding EventID 4662 and 5136, to use the 'winlog.event_data.SubjectUserName' as user.name and related.user

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/security.yml [hunk 1]
```diff
@@ -2551,7 +2551,7 @@ processors:
               "4742", "4743", "4744", "4745", "4746", "4747", "4748", "4749",
               "4750", "4751", "4752", "4753", "4754", "4755", "4756", "4757",
               "4758", "4759", "4760", "4761", "4762", "4763", "4764", "4767",
-              "4781", "4798", "4799", "4817", "4904", "4905", "4907", "4912"].contains(ctx.event.code)) {
+              "4781", "4798", "4799", "4817", "4904", "4905", "4907", "4912", "5136", "4662"].contains(ctx.event.code)) {
```
Comment (leehinman) [inline packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/security.yml:2554]: Can you add this to https://github.com/elastic/integrations/blob/main/packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml as well.  We want to keep these in sync.

Also can you add the ECS categorization fields for these 2 events to the params at https://github.com/elastic/integrations/blob/19ba18733e482303e2a26c040ce5a30014dd87ae/packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml#L9 and https://github.com/elastic/integrations/blob/19ba18733e482303e2a26c040ce5a30014dd87ae/packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/security.yml#L13

**Review by leehinman — CHANGES_REQUESTED** [review]
you will need to update the version number in `packages/windows/manifest.yml` as well.

File: packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/security.yml (hunk 1 above)
Comment (ChriZzn) [reply packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/security.yml:2570]: Hello, should be done now, Regards

**Comment by ChriZzn** [comment]
Any Updates on this?

**Comment by efd6** [comment]
Sorry this was dropped. Please fix the conflict in the system changelog.

**Comment by ChriZzn** [comment]
Hello, should be resolved now, Regards

**Review by efd6 — COMMENTED** [review]
Are you able to provide test cases for this? If not, let me know and I will construct ones from the MS docs ([here](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4662) and [here](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-5136), but real cases would be better).

**Comment by ChriZzn** [comment]
> Are you able to provide test cases for this? If not, let me know and I will construct ones from the MS docs ([here](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4662) and [here](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-5136), but real cases would be better).

Hi, im sorry i cant provide you test data, because they are from active environments...

**Comment by efd6** [comment]
Please apply the this patch file and then resolve the conflicts, [0001-add-tests.patch](https://github.com/elastic/integrations/files/13223199/0001-add-tests.patch).

**Comment by ChriZzn** [comment]
Hi i applied the patch and resolved the conflicts....

**Comment by efd6** [comment]
The author you've used to apply the patch is making the CLA checker unhappy. Would you please sign the CLA with that name/email address as well?

**Comment by ChriZzn** [comment]
> The author you've used to apply the patch is making the CLA checker unhappy. Would you please sign the CLA with that name/email address as well?

Hi, should be signed now with username 'cw', Regards

**Comment by efd6** [comment]
It will need to match exactly, including the email address.

**Comment by ChriZzn** [comment]
so

> It will need to match exactly, including the email address.

Resigned the CLA, please DM me the email/username combo if there are still Problems

**Review by efd6 — APPROVED** [review]

**Review by leehinman — APPROVED** [review]
LGTM

**Pull request merged**